src/plugins/en.json

```diff
@@ -19,7 +19,7 @@
     "navigateSectionLabel": "Navigate to a Specific Page (Out of 13)",
     "linkProjectText": "Link to GitHub project repository",
     "linkProjectAnchor": "https://github.com/canada-ca/aia-eia-js",
-    "linkDirective": "http://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32592#appC",
+    "linkDirective": "http://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32592",
```

Comment:
Looks like the anchor link is still used on the French side

Reply:
see that's why we do code reviews :)  thanks, I removed the anchor in French now.